.github/pull_request_template.md

```diff
@@ -1,21 +1,15 @@
-### What does this PR do?
-
-(Please set a descriptive PR title. Use this space for additional explanations.)
-
-### What does it look like?
-
-(If this is more than a textual change, a screenshot can help in some cases to speed up the review process.)
-
-### Any background context you can provide?
+<!---
+THIS CONTENT WILL BE PUBLIC ON THE HANDBOOK!
```

Comment:
Isn't it "too late" anyway once the branch has been pushed to this public repository?